netbox/dcim/tables/devices.py

```diff
@@ -466,6 +466,14 @@ class PowerPortTable(ModularDeviceComponentTable, PathEndpointTable):
             'args': [Accessor('device_id')],
         }
     )
+    maximum_draw = tables.TemplateColumn(
```

Comment:
We can use `netbox.tables.columns.TemplateColumn` instead of the stock column to automatically render the placeholder for empty values, which allows us to simplify the template code.